salt/orchestrate/aws/rds.sls

```diff
@@ -59,6 +61,22 @@ set_{{ name }}_master_password_in_vault:
 {% set master_pass = master_pass.data.value %}
 {% endif %}
 
+{# If engine is postgres and version is 10, then use major version only for
+   parameter group family. All others use major.minor. #}
+{% if engine == 'postgres' and engine_version.startswith('10') %}
+{% set fam_version = engine_version.partition('.')[0] %}
+{% else %}
+{% set fam_version = engine_version.rpartition('.')[0] %}
+{% endif %}
+{% set db_parameter_group_family = engine ~ fam_version %}
+
+
+create_{{ ENVIRONMENT }}_{{ name }}_parameter_group:
+  boto_rds.parameter_present:
```

Comment:
In addition to creating the parameter group we need to apply it to the RDS instance as well.

Reply:
I've just pushed a commit that will apply it in the `boto_rds.present` state.